src/platform/windows/input.cpp

```diff
@@ -536,12 +551,29 @@ namespace platf {
     auto &mi = i.mi;
 
     mi.dwFlags = MOUSEEVENTF_MOVE;
+    // mi.dwFlags = MOUSEEVENTF_MOVE | MOUSEEVENTF_MOVE_NOCOALESCE;  // TODO: Not sure if needed
```

Comment:
Before merging we should probably remove the commented out code.

Comment applies to the block above and a line in the `get_mouse_loc` function as well.

How were you testing this on Windows? Only via the CI workflows?

Reply:
At first yeah, but later I got it running on windows, but I can't continue implementing this on windows this month, as we discussed. I'll remove the comments and keep them in another branch to continue when I'm available again.